Dockerfile.dapper

```diff
@@ -21,15 +33,15 @@ RUN apt-get update -qq && apt-get install -y --no-install-recommends \
         docker-ce-cli=5:19.03.* \
     && rm -rf /var/lib/apt/lists/*
 ## install golangci
-RUN if [ "$(go env GOARCH)" = "amd64" ]; then \
+RUN if [ "${ARCH}" = "amd64" ]; then \
         curl -sL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b "$(go env GOPATH)/bin" v1.27.0; \
     fi
 ## install controller-gen
-RUN if [ "$(go env GOARCH)" = "amd64" ]; then \
+RUN if [ "${ARCH}" = "amd64" ]; then \
         GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1; \
     fi
 ## install ginkgo
-RUN if [ "$(go env GOARCH)" = "amd64" ]; then \
+RUN if [ "${ARCH}" = "amd64" ]; then \
         GO111MODULE=on go get github.com/onsi/ginkgo/ginkgo@v1.14.0; \
     fi
```

Comment:
can u please help to add `DAPPER_ENV` with GOPROXY setting, this will default to `direct` but allow the user to config if needed.